content/bn/docs/reference/glossary/cloud-provider.md

```diff
@@ -0,0 +1,27 @@
+---
+title: ক্লাউড প্রোভাইডার
+id: cloud-provider
+date: 2018-04-12
+short_description: >
+  একটি প্রতিষ্ঠান যা একটি ক্লাউড কম্পিউটিং প্ল্যাটফর্ম অফার করে।
+
+aka:
+- ক্লাউড সার্ভিস প্রোভাইডার
+tags:
+- কমিউনিটি
+---
+একটি ব্যবসা বা অন্য কোন প্রতিষ্ঠান যা একটি ক্লাউড কম্পিউটিং প্ল্যাটফর্ম অফার করে।
+
+<!--more-->
+
+ক্লাউড প্রোভাইডাররা, যাদের কখনও কখনও ক্লাউড সার্ভিস প্রোভাইডার (CSPs) বলা হয়,
+ক্লাউড কম্পিউটিং প্ল্যাটফর্ম বা সেবা প্রদান করে।
```

Comment:
সার্ভিস

Reply:
please check if my work is fitted to your liking. i will surely fix it again